app/screens/edit_profile/edit_profile.js

```diff
@@ -25,6 +26,7 @@ import AttachmentButton from 'app/components/attachment_button';
 import mattermostBucket from 'app/mattermost_bucket';
 import LocalConfig from 'assets/config';
 
+const MAX_SIZE = 20 * 1024;
```

Comment:
@enahum I thought this is 20KB and not 20MB, isn't it?

(I should not make a comment here since this is already closed but I came across this while reviewing a PR which corrects this value)